content/en/contribute/docs/style-guide.md

```diff
@@ -296,6 +296,11 @@ It is good practice for the image to link to the image file so that a larger ver
 |right-hand side|right hand side|
 |left-hand|left hand|
 
+### Where to place your images
+
+To ensure your images are loaded on the docsite in the right format,place them in a folder that is named similar to your .md file. For example in the scenario below, the image is linked in the style-guide.md file, thus it is placed in the style-guide folder.
+{{< figure src="where-to-place-images.png" link="where-to-place-images.png" >}}
```

Comment:
I wonder if we should be using proprietary image icons in this sample image. The `.md` files are associated with VS Code.

Reply:
I see what you mean, Ashley made this change. Is this okay?(https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#images)

Reply:
I'm not sure what I'm supposed to be looking at here.

Reply:
If it's about this image: https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/where-to-place-images.png

Then yes! No proprietary logos there!